src/handlers/metricsHandlers.js

```diff
@@ -0,0 +1,35 @@
+import ListMetrics from '../lib/ListMetrics';
+
+export class BucketsHandler {
```

Comment:
This could be one class `ListMetricsHandler`, you can pass the type `buckets/accounts` to the constructor to instantiate appropriate metrics.
The filename needs to reflect the classname for readability - `ListMetricsHandler.js`